src/table/resizer/styles.scss

```diff
@@ -89,10 +94,6 @@ th:not(:last-child) > .divider-disabled {
   }
 }
 
-th:last-child > .resizer:not(.is-visual-refresh) {
```

Comment:
this was some old rule that never applied (because the descendant of the th is `.resize-wrapper`, not `.resizer`.